File: app/src/main/java/com/neome/java/api/ent/entAside/sig/SigEntInfo.java
// neome.ai API - do not change
//

package com.neome.java.api.ent.entAside.sig;

import com.neome.java.api.ent.base.dto.DtoEntUserAvatar;
import com.neome.java.api.meta.base.Types.DemoAppId;
import com.neome.java.api.meta.base.Types.LanguageKey;
import com.neome.java.api.meta.base.Types.MediaIdAvatar;
import com.neome.java.api.meta.base.Types.TimeZoneKey;
import com.neome.java.api.nucleus.base.sig.SigVersion;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class SigEntInfo extends SigVersion
{
  public String about;

  @Nullable
  public MediaIdAvatar avatarId;

  public DemoAppId demoAppId;

  @Nullable
  public DtoEntUserAvatar[] entUserAvatarList;

  @Nullable
  public LanguageKey[] languageSet;

  public String name;

  public TimeZoneKey timeZone;
}

File: app/src/main/java/com/neome/api/ent/entAside/sig/SigEntInfo.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.ent.entAside.sig

import com.neome.api.meta.base.Types.DemoAppId
import com.neome.api.ent.base.dto.DtoEntUserAvatar
import com.neome.api.meta.base.Types.LanguageKey
import com.neome.api.meta.base.Types.MediaIdAvatar
import com.neome.api.nucleus.base.sig.SigVersion
import com.neome.api.meta.base.Types.TimeZoneKey

interface SigEntInfo : SigVersion
{
  val about: String
  val avatarId: MediaIdAvatar?
  val demoAppId: DemoAppId
  val entUserAvatarList: List<DtoEntUserAvatar>?
  val languageSet: List<LanguageKey>?
  val name: String
  val timeZone: TimeZoneKey
}